deploy/k8s.yaml

```diff
@@ -49,10 +49,10 @@ spec:
                 operator: Exists
       containers:
         # TODO: Update to an official image once the build is automated via GCB.
-      - image: gcr.io/vishnuk-cloud/node-termination-handler@sha256:66f68f10cb552cba66ebadd038c958d275e88b2dba59f242f6ab661fccfec0d7
+      - image: gcr.io/vishnuk-cloud/gke-node-termination-handler@sha256:d733806b403567478e8f4e772664c2f3896c6ac894c1f927c698513c26528448
         name: node-termination-handler
         command: ["./node-termination-handler"]
-        args: ["--logtostderr", "--exclude-pods=$(POD_NAME):$(POD_NAMESPACE)", "--kubeconfig=/var/lib/kubelet/kubeconfig", "-v=10"]
+        args: ["--logtostderr", "--exclude-pods=$(POD_NAME):$(POD_NAMESPACE)", "-v=10"] # "--kubeconfig=/var/lib/kubelet/kubeconfig"
```

Comment:
I personally don't like to commit commented out code. It's there in git history, so we can retrieve it later if we want.

But I wouldn't block on this.